src/docfx/Errors.cs

```diff
@@ -385,6 +378,9 @@ public static Error InvalidTocSyntax(SourceInfo? source)
             /// Behavior: ✔️ Message: ❌
             public static Error InvalidTocLevel(SourceInfo? source, int from, int to)
                 => new Error(ErrorLevel.Error, "invalid-toc-level", $"The toc level can't be skipped from {from} to {to}.", source);
+
+            public static Error DuplicateJoinTocName(SourceInfo<string> name)
```

Comment:
>DuplicateJoinTocName [](start = 32, length = 20)

where is this being used?

Reply:
more clean up to do after the refactoring